### DISCUSSION TITLE
Please help me with contributions (Ramen List) 😋

### DISCUSSION THREAD

**Discussion by Canppchk** [NAIST-SE/AttractingContributors#1086] [2024 Attraction Contributions]
I need help with my ramen list.

What is your guy's favorite recommended ramen in Japan?

Come to my fork:
https://github.com/Canppchk/AttractingContributors-Ramen

Add to my README file.

I look forward to your contributions

**Comment by Canppchk** [comment]
I have recommended my top 10 favorite ramen lists.

1.Kaidashi Chukasoba Kuramu (https://maps.app.goo.gl/tXfnERyHhpoxr1nh7)

2.Ramen house Mitsuba (https://maps.app.goo.gl/U1L6EwVU9dVcGcRN8)

3.Amanojaku (https://maps.app.goo.gl/AjHpqc1HWZabzRXB9)

4.麺舎 いっとう 奈良富雄店　(https://maps.app.goo.gl/PmGfPAhKti68f85n6)

5.アノラーメン製作所 (https://maps.app.goo.gl/aCUuG4S6QoeX1ZZ98)

6.Cho-tonkotsu Nodo 8 (https://maps.app.goo.gl/81ANqfaTSbTGSjHc9)

7.平成麺業 石橋店 (https://maps.app.goo.gl/fuJsFZQuLcyqqnTe7)

8.らーめん壷 東生駒店 (https://maps.app.goo.gl/tR9zWcHubJgqFHm26)

9.Kadoya Shokudo (https://maps.app.goo.gl/SfhoDP4U2AAMPYef6)

10.Taishio Soba Touka (https://maps.app.goo.gl/DexLGNczyMMfQ53e9)

**Comment by gill876** [comment]
I highly recommend アノラーメン製作所!
https://maps.app.goo.gl/LRjQcQ1bXohEMay2A

Please make a contribution to my project!
https://github.com/NAIST-SE/AttractingContributors/discussions/1089#discussion-6846489

**Comment by oyewaleoyebode** [comment]
Tonkotsu Ramen - Rich, creamy broth made from pork bones, often cooked for hours to extract maximum flavor.

**Reply by Canppchk** [reply]
Thanks!! and I already contribute you too.
